examples/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "erc725.js-examples",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "postinstall": "cd .. && npm link && cd ./examples && npm link erc725.js",
+    "fetchData": "node src/fetchData.js",
+    "getData": "node src/getData.js",
+    "encodeData": "node src/encodeData.js",
+    "decodeData": "node src/decodeData.js"
+  },
+  "dependencies": {
+    "erc725.js": "0.5.7",
+    "web3": "1.5.0"
+  },
+  "devDependencies": {
+    "isomorphic-fetch": "^3.0.0"
+  }
+}
```

Comment:
It's not yet working but it will be nice because we can then provide interactive examples very easily just by pointing Stackblitz to the example folder like this:
https://stackblitz.com/github/ERC725Alliance/erc725.js/tree/feature/simplify-api/examples

Reply:
Indeed that's a good idea to put the examples right in the repo.